src/components/screens/registerDelegate/form/index.js

```diff
@@ -0,0 +1,17 @@
+/* istanbul ignore file */
+import { connect } from 'react-redux';
+import { withTranslation } from 'react-i18next';
+import { getActiveTokenAccount } from '@utils/account';
+import Form from './form';
+
+const mapStateToProps = state => ({
+  account: getActiveTokenAccount(state),
+  transactions: state.transactions,
```

Comment:
This seems to have been passed into the component wrapper but it's not destructured or used in the `form.js` component